skupstina_django/scraper/db_utils.py

```diff
@@ -20,6 +23,8 @@ def parse_item_details(subject_details):
 
 def write_period_to_db(period_text: str, period_url: str):
     start_date, end_date = parse_date_range(period_text)
+    date_format = '%d.%b.%Y'
```

Comment:
Yeah, I wanted to maintain some of the formatting style that's online. Personally, I prefer DD-MMM-YYYY because it removes the ambiguity of "Is that date first or month first"?
(Stupid Americans to blame for it. If I had my way, all dates would be in ISO format :P)